packages/kit/types/private.d.ts

```diff
@@ -195,7 +195,7 @@ export interface RequestOptions {
 
 export interface RouteDefinition {
```

Comment:
aside: it's strange to me that this is private API - it's public because it's part of the public builder methods